metron-deployment/roles/metron-builder/tasks/main.yml

```diff
@@ -15,4 +15,7 @@
 #  limitations under the License.
 #
 ---
-hdp_repo_def: "http://public-repo-1.hortonworks.com/HDP/centos6/2.x/updates/2.5.0.0/hdp.repo"
+- name: Build Deployment Artifacts
+  local_action: shell cd {{ metron_build_dir }} && mvn clean package -DskipTests -P HDP-2.5.0.0,mpack,build-rpms
```

Comment:
Maybe something like...
```
- name: Build Deployment Artifacts
   local_action: shell cd {{ metron_build_dir }} && mvn clean package -DskipTests -P HDP-2.5.0.0,mpack,build-rpms
   become: false
   run_once: true
   ignore_errors: yes
   register: results

- fails: msg="Failed to build Metron... Is Docker Running?  <Link to README?>"
   when: results.rc != 0
```

Or could we splits the Metron build into multiple steps?  First do the `mvn clean package`.  Then build the RPMS in a second step (the step that adds the Docker dependency) , so we know more concretely when the issue is likely to be that Docker not installed or not running?